File: src/main/java/pepedevelopers/cursitu/model/dto/ExamDTO.java
package pepedevelopers.cursitu.model.dto;

import java.util.List;

public record ExamDTO(
  String date,
  String startTime,
  String endTime,
  String type,
  Integer totalStudents,
  List<TeacherSubmissionDTO> studentList,
  String assistment,
  Float note
) {}

File: src/app/models/dto/examDTO.ts
import { TeacherSubmissionDTO } from "./teacher-submissionDTO";

export interface ExamDTO {
  date: string | null;
  startTime: string | null;
  endTime: string | null;
  type: 'VIRTUAL' | 'PRESENCIAL';
  totalStudents: number;
  studentList: TeacherSubmissionDTO[];
  assistment: string;
  note: number;
}